packages/manager/modules/log-to-customer/src/live-tail/template.html

```diff
@@ -0,0 +1,82 @@
+<log-live-tail
+    data-log-keys="$ctrl.LOG_KEYS"
+    data-log-kind-api-url="$ctrl.logKindApiUrl"
+    data-log-api-url="$ctrl.logApiUrl"
+>
+    <description>
+        <span data-translate="logs_description"></span>
+        <a
+            data-ng-href="{{:: $ctrl.links.logInfo }}"
+            target="_blank"
+            rel="noopener"
+        >
+            <span data-translate="logs_info_link"></span>
+            <span
+                class="oui-icon oui-icon-external-link"
+                aria-hidden="true"
+            ></span>
+        </a>
+    </description>
+
+    <tile>
+        <h5 class="text-primary" data-translate="log_tile_section_title"></h5>
+        <oui-tile
+            data-heading="{{:: 'log_tile_title' | translate }}"
+            data-loading="$ctrl.loading"
+        >
+            <span data-translate="{{'log_tile_desc_' + $ctrl.service}}"></span>
+            <br />
+            <span
+                data-ng-if="$ctrl.streams.length === 0"
+                data-translate="log_tile_desc_create_account"
+            ></span>
+            <span
+                data-ng-if="$ctrl.streams.length > 0 && !$ctrl.hasAtLeastOneDataStream"
+                data-translate="log_tile_desc_create_stream"
+            ></span>
+
+            <a
+                data-ng-href="{{ :: $ctrl.links.logPlatformGuide}}"
+                target="_blank"
+                rel="noopener"
+            >
+                <span data-translate="log_data_platform_guide_link"></span>
+                <span
+                    class="oui-icon oui-icon-external-link"
+                    aria-hidden="true"
+                ></span>
+            </a>
+
+            <div class="my-2">
+                <a
+                    data-ng-if="$ctrl.streams.length === 0"
+                    class="oui-button oui-button_secondary"
+                    data-translate="log_button_create_account"
+                    data-ng-href="{{:: $ctrl.links.createAccount }}"
+                    target="_blank"
+                    rel="noopener"
+                >
+                </a>
+
+                <a
+                    data-ng-if="$ctrl.streams.length > 0 && !$ctrl.hasAtLeastOneDataStream"
```

Comment:
Is that condition really possible ? Having streams without having at least one ?

Reply:
According to last comment yes it is possible